### ISSUE TITLE
Using sequence as parameters

### ISSUE THREAD

**Issue by freud14-tm** [databricks/databricks-sql-python#290]
I know that using sequence as parameters works using the legacy inline parameters. However, it doesn't work with the native parameters. Is this somewhere on the roadmap to be implemented?

**Comment by susodapop** [comment]
> Is this somewhere on the roadmap to be implemented? 

Yes. For this to work, Databricks runtime needs to support `ARRAY` type SQL parameters. That feature is being designed right now.

> If so, when?

I don't have an ETA to share at this time. But I've passed this specific use-case to that team for consideration.

**Comment by freud14-tm** [comment]
Thank you very much @susodapop . Let me know of the development.

**Comment by susodapop** [comment]
From what I understand, the likely syntax for this construct in the future would be like this:

```sql
SELECT field FROM table WHERE field in (SELECT * FROM explode(:array_param))
```

...although it would be nice to have a better shorthand for this.

**Comment by nchammas** [comment]
Is support for native sequence parameters still waiting on Databricks Runtime support?

**Comment by freud14-tm** [comment]
Just a word of caution in case anyone was thinking on using this on a public-facing website or something.

**Comment by s-banach** [comment]
Did databricks fire everybody who works on useful stuff like SQL so they could go all-in on hosting LLMs, or what.
